Makefile

```diff
@@ -69,7 +69,7 @@ embedded-meta-test: build-debug
 
 stateless-test: build-debug
 	rm -rf ./_meta/
-	bash ./scripts/ci/ci-run-stateless-tests-standalone.sh
+	ulimit -c 1024 bash ./scripts/ci/ci-run-stateless-tests-standalone.sh
```

Comment:
Please add ulimit to the unit tests as well.

Reply:
Also, I am not sure if 1024 is enough, need to confirm further

Reply:
@PsiACE  I have tested the script locally and it works fine when I run the test with default OS maxfiles(which is 256 in maxos). 1024 should be enough for test.

Reply:
please test for unit-test

Reply:
when I ran the `make test` in my local machine,  the unit tests passed with the default maxfiles config(256 in macos). do we need to add ulimit to the UT? @PsiACE

Reply:
yes, we also need to add `ulimit -c 1024` to the UT

Reply:
On some machines, the default value (256) does not pass unit tests